zio-sql/jvm/src/main/scala/Sql.scala

```diff
@@ -217,7 +220,7 @@ trait Sql {
   def select[F, A, B <: SelectionSet[A]](selection: Selection[F, A, B]): SelectBuilder[F, A, B] =
     SelectBuilder(selection)
 
-  def deleteFrom[A](table: Table.Aux[A]): DeleteBuilder[A] = DeleteBuilder(table)
+  def deleteFrom[F[_], A, B](table: Table.Source.Aux[F, A, B]): DeleteBuilder[F, A, B] = DeleteBuilder(table)
```

Comment:
Could you try `table: Table.Source.Aux[_, A, _]` so we can avoid introducing `F` and `B`?

Reply:
The first parameter of Table.Source.Aux is higher kinded, so can't just use _. @adamgfraser suggested using a type lambda which I think could work but figured we don't really care about it being a Table.Source.Aux past a certain point, so could just widen it to just be a Table.Aux[A] instead. Still have a couple of extra parameters on `DeleteBuilder` but it's an intermediate type so don't think this should be an issue - the `Delete` type still only has one parameter which I think was the goal.  What do you think of this approach?